### PR TITLE
ci: add initial ci flow

### PR DESCRIPTION
**Pull request by peterdeme**
Add initial ci

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        with:
+          go-version: "1.18"
+
+      - uses: actions/checkout@v2
+
+      # - name: Test
+      #   env:
+      #     STREAM_KEY: ${{ secrets.STREAM_KEY }}
+      #     STREAM_SECRET: ${{ secrets.STREAM_SECRET }}
+      #   run: go test -v -race ./...
```
Comment (peterdeme) [inline .github/workflows/ci.yaml:28]: Can someone add the keys pls?

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+          GOOS=linux go build -o $PWD/linux ./cmd/stream-cli
+          GOOS=windows go build -o $PWD/windows.exe ./cmd/stream-cli
+
+      - name: Create release on GitHub
+        uses: ncipollo/release-action@v1
+        with:
+          body: ${{ env.CHANGELOG }}
+          tag: ${{ env.VERSION }}
+          token: ${{ secrets.GITHUB_TOKEN }}
+          artifacts: "darwin,linux,windows.exe"
```
Comment (peterdeme) [inline .github/workflows/release.yml:47]: I want to change this to [goreleaser](https://goreleaser.com/) a bit later, but let's just keep it this way for now

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (ferhatelmas) [reply .github/workflows/ci.yaml:28]: added

File: .github/workflows/release.yml (hunk 2 above)
Comment (ferhatelmas) [reply .github/workflows/release.yml:47]: sounds good

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (ferhatelmas) [reply .github/workflows/ci.yaml:28]: good to enable and merge

**Pull request merged**